package/src/utils/getResizedImageUrl.ts

```diff
@@ -25,11 +25,26 @@ export function getResizedImageUrl({
   url,
   width,
 }: GetResizedImageUrlParams) {
-  // Check if url belongs to cloudfront CDN
-  const isResizableUrl = url.includes('.stream-io-cdn.com');
-  if (!isResizableUrl || (!height && !width)) return url;
+  let parsedUrl;
```

Comment:
Why not just do this in the `try` block? something like

```js
  try {
  const parsedUrl = new URL(url);
  const originalHeight = parsedUrl.searchParams.get('oh');
  const originalWidth = parsedUrl.searchParams.get('ow');

  // If url is not from new cloudfront CDN (which offers fast image resizing), then return the url as it is.
  // Check for oh and ow parameters in the url, is just to differentiate between old and new CDN.
  // In case of old CDN we don't want to do any kind of resizing.
  const isResizableUrl = url.includes('.stream-io-cdn.com') && originalHeight && originalWidth;

  if (!isResizableUrl || (!height && !width)) return url;

  if (height) {
    parsedUrl.searchParams.set('h', `${PixelRatio.getPixelSizeForLayoutSize(Number(height))}`);
  }
  if (width) {
    parsedUrl.searchParams.set('w', `${PixelRatio.getPixelSizeForLayoutSize(Number(width))}`);
  }
  parsedUrl.searchParams.set('resize', `${resize}`);
  return parsedUrl.toString();
  } catch (error) {
    // There is some issue with the url.
    // Simply return the original url, there is no need to break the app for this.
    console.warn(error);

    return url;
  }


```

Reply:
In this case it doesn't matter much since its a small function. And it does avoid defining a variable and then assigning inside try block, so have done it for this case.

But in general I don't like the idea of wrapping entire code un-necessarily inside a try-catch block, which suppresses the thrown error (as in this case)